app/controllers/valets_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ValetsController < ApplicationController
+
+  def update
+    @valet = Valet.find_by_id(params[:id])
+    if @valet.nil?
+      render json: {message: "404 Not Found"}
+    else
+      @valet.update(valet_params)
```

Comment:
[update](http://apidock.com/rails/ActiveRecord/Persistence/update) should handle the case that it can't update in the database too